linear_operator/test/linear_operator_test_case.py

```diff
@@ -606,7 +615,11 @@ def test_add_jitter(self):
         linear_op = self.create_linear_op()
         evaluated = self.evaluate_linear_op(linear_op)
 
-        res = linear_operator.add_jitter(linear_op, 0.4).to_dense()
+        try:
```

Comment:
Why are these try/excepts needed?

Reply:
Oops. Removed. This was some debugging I added to track down a function signature error.